Makefile

```diff
@@ -1374,6 +1388,31 @@ mpi_f08_test:
 	$(if $(findstring 1,$(MPAS_MPI_F08)), $(eval MPI_F08_MESSAGE = "Using the mpi_f08 module."), )
 	$(if $(findstring 1,$(MPAS_MPI_F08)), $(info mpi_f08 module detected.))
 
+musica_fortran_test:
+	@#
+	@# Create a Fortran test program that will link against the MUSICA library
+	@#
+	$(info Building a simple program with the MUSICA-Fortran library...)
+	$(eval MUSICA_FORTRAN_TEST := $(shell $\
+		printf "program test_musica_fortran\n$\
+						&   use musica_util, only : string_t\n$\
+						&   use musica_micm, only : get_micm_version\n$\
+						&   type(string_t) :: version_string\n$\
+						&   version_string = get_micm_version()\n$\
+						&   print *, \"MUSICA support is available. MICM version: \", version_string%%value_\n$\
+						end program test_musica_fortran\n" | sed 's/&/ /' > test_musica_fortran.f90; $\
+		$\
+		$(FC) $(MUSICA_FCINCLUDES) $(MUSICA_FFLAGS) test_musica_fortran.f90 -o test_musica_fortran.x $(MUSICA_LIBS) > /dev/null 2>&1; $\
+		musica_fortran_status=$$?; $\
+		rm -f test_musica_fortran.f90 test_musica_fortran.x; $\
+		if [ $$musica_fortran_status -eq 0 ]; then $\
+			printf "1"; $\
+		else $\
+			printf "0"; $\
+		fi $\
+	))
+	$(if $(findstring 0,$(MUSICA_FORTRAN_TEST)), $(error Could not build a simple test program with MUSICA-Fortran))
+	$(if $(findstring 1,$(MUSICA_FORTRAN_TEST)), $(eval CHEMISTRY_MESSAGE = "MUSICA-Fortran library available."), )
```

Comment:
So that we don't set `CHEMISTRY_MESSAGE` (perhaps renamed `MUSICA_MESSAGE`) in two different places, here it might suffice to report the MUSICA library version that was found?